server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -392,10 +404,16 @@ module.exports = {
         action: caSubmitAddressReview.getCaAction({ decisions, tasks }),
         visible: !bassReferralNeeded && allowedTransition === 'caToRo',
       },
+      {
+        title: 'Resubmit to DM',
+        label: caRereferDm.getLabel(),
+        action: caRereferDm.getCaAction(),
+        visible: !['caToDm', 'caToDmRefusal', 'caToRo'].includes(allowedTransition) && dmRefused !== undefined,
+      },
       {
         title: 'Create licence',
         action: createLicence.getCaAction({ decisions, tasks, stage }),
-        visible: validAddress && !['caToDm', 'caToDmRefusal', 'caToRo'].includes(allowedTransition),
+        visible: validAddress && !['caToDm', 'caToDmRefusal', 'caToRo'].includes(allowedTransition) && !dmRefused,
```

Comment:
not quite sure why this needs to change?

Reply:
Originally then Continue button was displayed irrespective of whether the DM had approved or Rejected HDC. 
But now a decision has been made to show it but only if HDC has been approved. 
I thought  there are 3 potential states: 
1. DM hasn't yet decided - the scenario where the ca task list has been reached prior to journey ever reaching the DM and so so the dmRefused would be undefined
2: DM has approved
3: DM has rejected. 
Hence with !dmRefused, the Create Licence would only display if DM has decided and the decision is Yes to HDC.

Reply:
I'm not sure `dmRefused` is ever undefined. 

`getDmApproval` in `licenceStatus`:
```    
    approved: decision === 'Yes',
    refused: decision === 'No',
```

Would it make sense to hand this off `dmApproved` instead or even `approved`? 

`approved: dmApproval.approved && !caRefusal.refused`

Might be worth checking with Peter?